Aquality.Selenium.Template/Aquality.Selenium.Template.SpecFlow/Hooks/BrowserHooks.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Aquality.Selenium.Browsers;
+using TechTalk.SpecFlow;
+
+namespace Aquality.Selenium.Template.Glue.Hooks
```

Comment:
'glue' not so readable as 'steps' or 'stepdefinitions'
perhaps better to use 'steps'?

Reply:
By mistake, copied from another solution